ueventd.rc

```diff
@@ -13,10 +13,7 @@
 /sys/bus/iio/devices/iio:device* scan_elements/in_anglvel_y_en 0660 system system
 /sys/bus/iio/devices/iio:device* scan_elements/in_anglvel_z_en 0660 system system
 /sys/bus/iio/devices/trigger* 	 name 0660 system system
-/sys/class/backlight*		 brightness 0664 system system
-/sys/class/backlight*		 max_brightness 0664 system system
```

Comment:
Aren't ueventd.rc rules applied *only* for device files exposed via uevents? From some tests with real HW that is what it looked like, and those /sys/class/backlight/*/brightness files were not getting the correct permissions set. In particular, brightness wasn't being set RW.